doc/sphinx/user/methods/constitutive-laws/index.md

```diff
@@ -0,0 +1,21 @@
+(sec:methods:constitutive-laws)=
+# Constitutive Laws (Rheology)
+
+
+:::{toctree}
+constant-viscosity.md
```

Comment:
I don't know if it matters for now, but the constant-viscosity.md file is missing.